doc/api_methods.md

```diff
@@ -142,15 +142,15 @@ layerSource | contains information about the layer. It can be specified in multi
 options |
 --- | ---
 &#124;_ https | force https
-&#124;_ refreshTime | if is set, the layer is refreshed each refreshTime milliseconds.
-&#124;_ infowindow | set to false if you want to disable the infowindow (enabled by default).
+&#124;_ refreshTime | if is set, the layer is refreshed each refreshTime milliseconds.<br/><br/>For a code example of refreshTime, see ["layer_definition.spec.js"](https://github.com/CartoDB/cartodb.js/blob/60285edb65f6575c8a34799fde55308751251c5f/test/spec/geo/layer_definition.spec.js).
```

Comment:
I don't think this is a good reference :(. If examples are going to be part of the documentation I think you should prepare some stable and reliable examples that we can point to (or include the code snippet as part of the docs).